services/order.py

```diff
@@ -0,0 +1,41 @@
+from datetime import datetime
+
+from django.db.models import QuerySet
+
+from django.db import transaction
+
+from db.models import Order, Ticket, User, MovieSession
+
+
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: datetime = None
+) -> Order:
+    with transaction.atomic():
+        user = User.objects.get(username=username)
+        order = Order.objects.create(user=user)
+        if date:
+            Order.objects.filter(id=order.id).update(created_at=date)
+        tickets_instances = []
+        for ticket in tickets:
+            movie_session = MovieSession.objects.get(
+                id=ticket["movie_session"]
+            )
```

Comment:
The `create_order` function does not handle the case where a movie session with the specified ID does not exist. Consider using a try-except block to catch `MovieSession.DoesNotExist` exceptions and handle them appropriately.